ftl/node/benchmark/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM gcr.io/google-appengine/python
+RUN pip install google-cloud
+RUN apt-get update
```

Comment:
These three lines should all be on the same line: RUN apt-get update && apt-get -y install nodejs npm && apt-get clean && \
    rm /var/lib/apt/lists/*_*

(from here: https://github.com/GoogleCloudPlatform/base-images-docker/blob/35e20794b1770f3bb7e585e0b33f8bdab6827643/ubuntu/Dockerfile.ubuntu)

Although we probably want a specfic version of nodejs/npm, so it might be easier to use the ubuntu/nodejs image we build over here: https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/ftl/BUILD#L100

It should already have python.